### ISSUE TITLE
test_data.npz no find?

### ISSUE THREAD

**Issue by wildBigPanda** [hechmik/voxceleb_enrichment_age_gender#4]
Hi，author，thanks for great work！I want to test your pre-train model，Can you give me your test set file test_data.npz？
I try to creation it , but fail.
I just need mfcc age test_data.npz，thanks a lot!

**Comment by hechmik** [comment]
Hi linyongye, sorry for the huge delay, I did not get any notification.
Unfortunately we aren't the original data provider, therefore in order to avoid any copyright issue we prefer to not share the original data as well as any derivative info. I strongly suggest to get in touch with the VoxCeleb team and then follow the steps described in the paper and in the various README
